assets/src/block-editor/helpers/index.js

```diff
@@ -484,7 +483,7 @@ AmpNoloadingToggle.propTypes = {
  * Get AMP Lightbox toggle control.
  *
  * @param {Object} props Props.
- * @return {ReactElement} Element.
+ * @return {JSX.Element} Element.
```

Comment:
Theoretically you could have also done `@return {import('react').ReactElement} Element.`

Reply:
Hmm yea, I think that would require us disabling the `jsdoc/valid-types` ESLint rule though. We're currently using the [`@wordpress/eslint-plugin/recommended-with-formatting`](https://github.com/WordPress/gutenberg/blob/30744d0beb834c7f04a32bf1a2e6821c5e6bdfe5/packages/eslint-plugin/configs/recommended-with-formatting.js) plugin which in turn [adds `JSX`](https://github.com/WordPress/gutenberg/blob/30744d0beb834c7f04a32bf1a2e6821c5e6bdfe5/packages/eslint-plugin/configs/jsdoc.js#L100) as a valid type.